modules/usertracking-impl/src/main/java/org/opencastproject/usertracking/impl/UserSessionImpl.java

```diff
@@ -54,17 +57,17 @@
   private String sessionId;
 
   @Lob
-  @Column(name = "user_id", length = 255)
+  @Column(name = "user_id")
   @XmlElement(name = "userId")
   private String userId;
 
   @Lob
-  @Column(name = "user_ip", length = 255)
+  @Column(name = "user_ip")
```

Comment:
The field `user_ip` is generated as `LONGTEXT` instead of `VARCHAR(255)`

Reply:
Fixed. Need to verify.